### ISSUE TITLE
Between Composition Rule does not work for TPC-H Q19

### ISSUE THREAD

**Issue by Bouncner** [hyrise/hyrise#2274]
In TPC-H Q19, the predicates `[..] AND l_quantity >= [QUANTITY1] AND l_quantity <= [QUANTITY1] + 10 [..]` are not rewritten to a between predicate.

<img width="991" alt="image" src="https://user-images.githubusercontent.com/1745857/100647649-1baa4800-3340-11eb-8237-0246ff1a27c5.png">

**Comment by mrks** [comment]
I guess it is because the `PredicateSplitUpRule` has not yet split up disjunctions yet.

**Comment by mrks** [comment]
Should be save to move the BetweenCompositionRule behind the PredicateSplitUpRule as PredicatePlacementRule is executed a second time, anyway.

**Comment by julianmenzler** [comment]
> Should be safe to move the BetweenCompositionRule behind the PredicateSplitUpRule as PredicatePlacementRule is executed a second time, anyway.

This seems to work. 
I will make a new PR and measurements after #2284 has been merged.
